libyul/backends/evm/EVMDialect.cpp

```diff
@@ -531,6 +539,7 @@ BuiltinFunctionForEVM EVMDialect::createVerbatimFunction(size_t _arguments, size
 		1 + _arguments,
 		_returnVariables,
 		SideEffects::worst(),
+		ControlFlowSideEffects{},
```

Comment:
Oh, just noticed this now - looks like this PR actually uncovered a pre-existing bug. I think this should be `ControlFlowSideEffects{true, true, true}` because `verbatim` can do virtually anything.

Can you submit a fix for this in a separate PR? This will need a changelog entry and we need to think how serious this is in practice. It's not relevant to EOF though, so I don't want all this to block merging of this PR.

Reply:
https://github.com/ethereum/solidity/pull/15542